cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -521,11 +521,10 @@ def render_files():
     render('kube-defaults.defaults', '/etc/default/kube-defaults', context)
 
     # when files change on disk, we need to inform systemd of the changes
-    try:
-        check_call(['systemctl', 'daemon-reload'])
-    except CalledProcessError:
-        # we failed, chances are no changes were made. so assume this is fine
-        pass
+    call(['systemctl', 'daemon-reload'])
+    call(['systemctl', 'enable', 'kube-apiserver'])
```

Comment:
You are not calling the new method that you created. Clearly an oversight. But change these three lines to:
```python
_enable_services(['kube-apiserver', 'kube-controller-manager', 'kube-scheduler'])
```